pallets/settlement/src/lib.rs

```diff
@@ -89,17 +89,32 @@ pub trait Trait:
     type MaxLegsInAnInstruction: Get<u32>;
 }
 
+#[derive(
+    Encode, Decode, Clone, Debug, PartialEq, Eq, VecU8StrongTyped, Default, PartialOrd, Ord,
+)]
+pub struct Base64Vec(pub Vec<u8>);
```

Comment:
Let's define this in primitives and import it here and in confidential-asset pallet.

Reply:
Sure, will do it tomorrow.